### DISCUSSION TITLE
Persistent Issue with GitHub Education Verification – Seeking Community Assistance

### DISCUSSION THREAD

**Discussion by sharklatan** [community/community#77750] [GitHub Education]
### Hi everyone,

Hello GitHub Community,

I hope this message finds you well. I'm reaching out to share an ongoing issue I've encountered while attempting to verify my application for the GitHub Education program. I believe this problem may be affecting others, and I'm seeking assistance or insights from the community.

I've tried multiple documents for verification, encountering issues with each attempt. Despite my efforts to address concerns such as my name and billing information, a persistent error related to location prevents me from progressing. The error message reads:

"Your application cannot be reviewed until you fix the following:
You appear not to be near any campus location for the school you have selected. Your academic affiliation documentation must indicate virtual learning if you are remote."

It's crucial to note that I selected my educational institution, "Preparatoria en línea SEP," which translates to "High School Online - SEP" in English, as it is exclusively online. The system continues to indicate that I am not near any campus location, which is consistent with the fact that the institution is online and does not have a physical campus.

Furthermore, it's important to highlight that my institution does appear in the GitHub system, but there seems to be a misalignment or error in the information.

Additionally, the academic institution is part of the government of Mexico. In this context, online educational institutions typically do not specify "online" in their documentation, as this modality is implicit in their name, as in the case of "Preparatoria en línea SEP."

I'm reaching out to seek guidance from the community. Has anyone else faced similar challenges or can offer insights into resolving this issue?

Thank you for your time and any assistance the community can provide.

Best regards,
sharklatan

**Comment by dakshithadissanayaka** [comment]
It seems like there might be a discrepancy in the GitHub Education program's verification process for online institutions. The error message suggests that the system is expecting students from online institutions to explicitly state their virtual learning status in their academic affiliation documentation. However, as you've mentioned, this information is often implicit in the name of the institution itself, especially for government-run online schools like "Preparatoria en línea SEP."

To address this issue, I would recommend the following steps:

1. **Contact GitHub Education Support:** Reach out to GitHub Education support directly and explain the situation. They may be able to manually review your application and verify your enrollment status based on the available information.

2. **Provide Additional Documentation:** If possible, gather any additional documentation that supports your enrollment at "Preparatoria en línea SEP," such as official transcripts, student ID cards, or online enrollment confirmation emails.

3. **Highlight Government Affiliation:** Emphasize the fact that your institution is part of the government of Mexico, as online educational institutions under government auspices may have different documentation standards.

4. **Seek Community Support:** Engage with the GitHub Education community on forums or social media platforms to see if others have encountered similar issues and if they have found solutions.

5. **Raise Awareness:** Consider creating a public post or discussion thread on GitHub Education's forums or social media channels to bring attention to this issue and potentially encourage a review of the verification process for online institutions.

By taking these steps, you can increase your chances of getting your application verified and gaining access to the benefits of the GitHub Education program. Remember to be patient and persistent, and don't hesitate to seek assistance from both GitHub Education support and the wider community.

**Comment by krittaprot** [comment]
I am having the same issue and have wasted 1-2 hours on this...
I got an automatic rejection email after submitting evidence that I am currently in an online master's program.
I have no ideas how to proceed from here.

**Comment by maueroats** [comment]
Title should be something like "Online education: unable to verify as student".
